### PR TITLE
Propose versioning types explanations

### PR DESCRIPTION
**Pull request by samuelFain**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Proposing explanations for the versioning types.
Would appreciate your input.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/documentation/release-notes.md [hunk 1]
```diff
@@ -13,7 +13,14 @@ demisto-sdk update-release-notes -i [Changed pack path] -u [major|minor|revision
 
 **Please note:** Changes that have not been committed are not detected automatically by the ***update-release-notes*** command.
 
-This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag).
+This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag), the versioning format is as follows: MAJOR.MINOR.REVISION.
+
+| versioning type | when to use |
+|---|---|
+| major | when you make a new version of the pack or one of its integrations. |
+| minor | when you add functionality (for example: mirroring, fetching incidents or indicators). |
```
Comment (tkatzir) [inline docs/documentation/release-notes.md:21]:
```suggestion
| minor | when you add functionality (for example: mirroring, fetching incidents, or indicators). |
```

File: docs/documentation/release-notes.md [hunk 2]
```diff
@@ -13,7 +13,14 @@ demisto-sdk update-release-notes -i [Changed pack path] -u [major|minor|revision
 
 **Please note:** Changes that have not been committed are not detected automatically by the ***update-release-notes*** command.
 
-This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag).
+This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag), the versioning format is as follows: MAJOR.MINOR.REVISION.
+
+| versioning type | when to use |
+|---|---|
+| major | when you make a new version of the pack or one of its integrations. |
```
Comment (tkatzir) [inline docs/documentation/release-notes.md:20]:
```suggestion
| major | when you make a new version of the pack, or one of its integrations. |
```

File: docs/documentation/release-notes.md [hunk 3]
```diff
@@ -13,7 +13,14 @@ demisto-sdk update-release-notes -i [Changed pack path] -u [major|minor|revision
...
+| minor | when you add functionality (for example: mirroring, fetching incidents or indicators). |
+| revision | when you add new content items or commands, and when updating such. |
```
Comment (tkatzir) [inline docs/documentation/release-notes.md:22]: If you add many commands, then you'd update the minor version.
```suggestion
| revision | when you add new content items or a few commands, and when updating such. |
```

**Review by tkatzir — COMMENTED** [review]
Looks good. A few minor ( ;-) ) comments.

File: docs/documentation/release-notes.md (hunk 2 above)
Comment (ShacharKidor) [inline docs/documentation/release-notes.md:20]:
```suggestion
| major | when you make a new version of the pack, or one of its integrations (for example rewrite). |
```

**Review by ShacharKidor — APPROVED** [review]
cool :)
Let's add a tech writer (from the tech doc team) for a review.

File: docs/documentation/release-notes.md [hunk 4]
```diff
@@ -13,7 +13,14 @@ demisto-sdk update-release-notes -i [Changed pack path] -u [major|minor|revision
...
 
-This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag).
+This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag), the versioning format is as follows: MAJOR.MINOR.REVISION.
+
+| versioning type | when to use |
+|---|---|
+| major | when you make a new version of the pack, or one of its integrations (For example, a rewrite of integration, or a vast change in it.). |
+| minor | when you add functionality (for example: mirroring, fetching incidents, or indicators). |
+| revision | when you add new content items or a few commands, and when updating such. |
+| documentation | when adding or updating documentation. |
```
Comment (ShacharKidor) [inline docs/documentation/release-notes.md:23]:
```suggestion
| major | When you make a new version of the pack, or one of its integrations (for example, a rewrite of integration, or a vast change in it). |
| minor | When you add functionality (for example mirroring, fetching incidents, or indicators). |
| revision | When you add new content items or a few commands, and when updating such. |
| documentation | When adding or updating documentation. |
```

File: docs/documentation/release-notes.md [hunk 5]
```diff
@@ -13,7 +13,14 @@ demisto-sdk update-release-notes -i [Changed pack path] -u [major|minor|revision
 
 **Please note:** Changes that have not been committed are not detected automatically by the ***update-release-notes*** command.
 
-This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag).
+This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag), the versioning format is as follows: MAJOR.MINOR.REVISION.
```
Comment (ShirleyDenkberg) [inline docs/documentation/release-notes.md:16]:
```suggestion
This command automatically updates the *currentVersion* found in the *pack_metadata.json* file according to the update version (as denoted by the *-u* flag). The versioning format is as follows: MAJOR.MINOR.REVISION.
```

Comment (samuelFain) [reply docs/documentation/release-notes.md:16]: Apply suggestions from docs review

**Pull request merged**